programs/launcher/main.cpp

```diff
@@ -397,6 +411,7 @@ launcher_def::initialize (const variables_map &vmap) {
         exit (-1);
       }
     }
+    add_enable_stale_production = true;
```

Comment:
Why is this true only if mode is passed?

Reply:
I only want this to be true when we are starting a new testnet from scratch. Otherwise it should use --replay or --resync.